docs/README.md

```diff
@@ -173,50 +173,49 @@ $ ARCH=aarch64 scripts/mk.image
+|`LOGDNA_HOSTNAME`|`http.params.hostname`|The hostname metadata to attach to lines forwarded from this agent||
+|`LOGDNA_IP`|`http.params.ip`|The IP metadata to attach to lines forwarded from this agent||
+|`LOGDNA_TAGS`|`http.params.tags`|Comma separated list of tags metadata to attach to lines forwarded from this agent||
+|`LOGDNA_MAC`|`http.params.mac`|The MAC metadata to attach to lines forwarded from this agent||
+|`LOGDNA_LOG_DIRS`<br>**Deprecated**: `LOG_DIRS`|`log.dirs[]`|List of folders to recursively monitor for log events|`/var/log/`|
+|`LOGDNA_EXCLUSION_RULES`<br>**Deprecated**: `LOGDNA_EXCLUDE`|`log.exclude.glob[]`|List of glob patterns to exclude files from monitoring <sup>1</sup>|`/var/log/wtmp,/var/log/btmp,/var/log/utmp,` <br>`/var/log/wtmpx,/var/log/btmpx,/var/log/utmpx,` <br>`/var/log/asl/**,/var/log/sa/**,/var/log/sar*,` <br>`/var/log/tallylog,/var/log/fluentd-buffers/**/*,` <br>`/var/log/pods/**/*`|
+|`LOGDNA_EXCLUSION_REGEX_RULES`<br>**Deprecated**: `LOGDNA_EXCLUDE_REGEX`|`log.exclude.regex[]`|List of regex patterns to exclude files from monitoring||
+|`LOGDNA_INCLUSION_RULES`<br>**Deprecated**: `LOGDNA_INCLUDE`|`log.include.glob[]`|List of glob patterns to includes files for monitoring <sup>1</sup>|`*.log`|
+|`LOGDNA_INCLUSION_REGEX_RULES`<br>**Deprecated**: `LOGDNA_INCLUDE_REGEX`|`log.include.regex[]`|List of regex patterns to include files from monitoring||
+|`LOGDNA_LINE_EXCLUSION_REGEX`|`log.line_exclusion_regex[]`|List of regex patterns to exclude log lines. When set, the Agent will NOT send log lines that match any of these patterns.||
+|`LOGDNA_LINE_INCLUSION_REGEX`|`log.line_inclusion_regex[]`|List of regex patterns to include log lines. When set, the Agent will send ONLY log lines that match any of these patterns.||
+|`LOGDNA_REDACT_REGEX`|`log.line_redact_regex`|List of regex patterns used to mask matching sensitive information (such as PII) before sending it in the log line.||
+|`LOGDNA_JOURNALD_PATHS`|`journald.paths[]`|List of paths (directories or files) of journald paths to monitor||
+|`LOGDNA_LOOKBACK`|`log.lookback`|The lookback strategy on startup|`none`|
+|`LOGDNA_K8S_STARTUP_LEASE`|`startup.option`|Determines whether or not to use K8 leases on startup|`never`|
+|`LOGDNA_USE_K8S_LOG_ENRICHMENT`|`log.use_k8s_enrichment`|Determines whether the agent should query the K8s API to enrich log lines from other pods.|`always`|
+|`LOGDNA_LOG_K8S_EVENTS`|`log.log_k8s_events`|Determines whether the agent should log Kubernetes resource events. This setting only affects tracking and logging Kubernetes resource changes via watches. When disabled, the agent may still query k8s metadata to enrich log lines from other pods depending on the value of `LOGDNA_USE_K8S_LOG_ENRICHMENT` setting value.|`never`|
+|`LOGDNA_LOG_REPORTER_METRICS`|n/a|Determines whether or not metrics usage statistics is enabled.|`never`|
```

Comment:
I wonder if I should make these *n/a* as well since they are only k8s (not set in yaml)?